src/frameworks/astro/utils.ts

```diff
@@ -10,15 +12,31 @@ export function getBootstrapScript() {
 
 export async function getConfig(cwd: string) {
   const astroDirectory = dirname(require.resolve("astro/package.json", { paths: [cwd] }));
-  const { openConfig }: typeof import("astro/dist/core/config/config") = await dynamicImport(
-    join(astroDirectory, "dist", "core", "config", "config.js")
-  );
-  const logging: any = undefined; // TODO figure out the types here
-  const { astroConfig: config } = await openConfig({ cmd: "build", cwd, logging });
+  const version = getAstroVersion(cwd);
+
+  let config;
+  if (gte(version!, "2.9.7")) {
+    const { resolveConfig } = await dynamicImport(
+      join(astroDirectory, "dist", "core", "config", "config.js")
```

Comment:
nit* Seems like this line and L26 is identical - do we want to extract this out as a variable so we can more easily identify difference between how we handle versions <2.9.7 and >=2.9.7?